### PR TITLE
Prepare package for PyPI

### PR DESCRIPTION
**Pull request by sophiadavis**
https://pypi.python.org/pypi/amqpconsumer/1.0

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    long_description='Python module to facilitate listening for and acting on AMQP events.',
+    url='https://github.com/ByteInternet/amqpconsumer',
+    author='Byte B.V.',
+    author_email='tech@byte.nl',
     license='3-clause BSD',
-    author='Maarten van Schaik',
-    author_email='maarten@byte.nl',
-    description='RabbitMQ event listener',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```
Comment (gertjanol) [inline setup.py]: That's a bold statement ;)

Comment (sophiadavis) [inline setup.py]: Yeah... but it is technically in use in production!

Comment (allardhoeve) [inline setup.py]: Maybe we should use a valid email address :-)

**Comment by sophiadavis** [comment]
@gertjanol 
Once the [hn-control](https://github.com/ByteInternet/hypernode-control/pull/1632) change is merged, I think we're good to merge this pr and rename the repo to `amqpconsumer`.

In the meantime, @allardhoeve added a Scrutinizer Code Quality badge. I guess we'll also need to 'scrutinize' the renamed repo?

**Pull request merged**